NtupleProducer/plugins/HTauTauNtuplizer.cc

```diff
@@ -796,7 +796,7 @@ class HTauTauNtuplizer : public edm::EDAnalyzer {
   double st_mult;
   unsigned _MC_pdf_first_idx;
   unsigned _MC_pdf_last_idx;
-  unsigned _MC_QCDscale_indices[7];
+  unsigned _MC_QCDscale_indices[7] = {};
```

Comment:
this could be ```std::vector<unsigned> _MC_QCDscale_indices(7, 0);```

Reply:
After a quick discussion, we decided I'll update this on my side for efficiency.